index.html

```diff
@@ -0,0 +1,46 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+  <title>Library</title>
+</head>
+<body class="bg-info">
+
+  <div id = "alertText" class="w-100 bg-danger">
+    <div class = 'w-fit-content m-auto'>
+      Book Title, Author Name and Total Pages must be filled out
+    </div>
+  </div>
+
+  <h1 class="w-fit-content mx-auto mt-5 font-weight-bold">Library</h1>
+  <button id = "add-btn" type="button" class="btn btn-primary ml-5 mt-5">Add a Book</button>
+
+  <div class="container mt-5" id="conatiner">
+    <div class="row" id="demo"> </div>
+  </div>
+
+  <div id = 'display'>
+    <h2 class="w-fit-content mx-auto mb-3">Add a New Book</h2>
+      <form id = 'bookForm' class="w-50 m-auto">
+        <div class="form-group">
+          <label for="bookName">Book Title</label>
+          <input type="text" id="bookName" class="form-control">
+        </div>
+        <div class="form-group">
+          <label for="authorName">Author Name</label>
+          <input type="text" id="authorName" class="form-control">
+        </div>
+        <div class="form-group">
+          <label for="totalPages">Total Pages</label>
+          <input type="text" id="totalPages" class="form-control">
+        </div>
+        <input type="submit" value="Submit" class="btn btn-primary btn-block">
+      </form> 
+  </div>
+
+  <script src="index.js"></script>
+</body>
+</html>
```

Comment:
## Add a descriptive README 
- [ ] Please enure the project has a descriptive `README.md` before we proceed. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).

Reply:
## Improve the live link in the README
- Great job working on the README. it looks good.
- [ ] One final change, do you mind working on the live link to actually point to the source where the page is hosted live for reviewers to actually get to test the code in production as well? Below is what we currently have 👇🏾 
![library-live](https://user-images.githubusercontent.com/22345961/111680973-d4cf5800-8833-11eb-88ea-b5af8edcaca5.png)